### DISCUSSION TITLE
Changing shell

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1153] [Q&A]
Is it possible to switch from fisch to zsh and vice versa afterwards

**Comment by kRHYME7** [comment]
Change shell then also get the configuration from hyprdots? or just changing shell? 

 ``` chsh ``` command can do the trick if you have the fish installed already.

**Reply by dieBakterie** [reply]
The Shell with the configs ig?

**Reply by kRHYME7** [reply]
Ok let me add a wrapper for that on Hyprdots CLI laters


basically it should do this 

+ install ``` fish ``` 
+ copy all the configs that need fish 
+ then run chsh -s /usr/bin/fish

**Reply by dieBakterie** [reply]
Yeah but actually i'm using fish, so it's more Like changing from fish to zsh. And thanks for this great Tool. Oh i have not understand until know what you meant with `chsh`.

**Reply by kRHYME7** [reply]
Oh! I almost forgot about this can you open an issue here ? 

https://www.google.com/search?client=firefox-b-d&q=hyprdots-ctl-git

**Reply by dieBakterie** [reply]
You meant for changing the Shell?

**Reply by kRHYME7** [reply]
yup

**Reply by dieBakterie** [reply]
Done and there is actually no Script for restoring fish in hyprdots and only for zsh right?

**Reply by kRHYME7** [reply]
> Done and there is actually no Script for restoring fish in hyprdots and only for zsh right? 

fish configs only needs a dependency ``` lsd ``` and fish 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/4d4d859b-fdd4-4336-9e7e-28f8971b7025)


is the lsd and fish are installed then this files will be imported to your configs

**Reply by dieBakterie** [reply]
OK this Projekt ist Just so huge that i haven't got through it and btw i can't click to Open an Applications tough it works for the Tabs etc, If not i'll Open an issue for this. And is it a brave Problem but i can't get to the Last entry in the Tabs Overview an for an Applications is the Icon Missing though it's there in the Applications Launcher

**Reply by kRHYME7** [reply]
Sorry I did not catch up. Can you elaborate more?

**Reply by dieBakterie** [reply]
In the rofi menu/applications launcher(SUPER + A) i can't click apps/entrys to open them i need to press enter but i can open the left entries (menu_style_1) by clicking(nvm it's just double click). Brave(flatpak) and youtube-music-bin(aur) don't have an icon there but in applications menu they've. Why is "screenshot" described "Frozen Screen"?
![image](https://github.com/prasanthrangan/hyprdots/assets/50206827/90ca57ef-e1b7-4beb-b863-7f1834174982)
That's what i mean and if that's an flatpak or electron or x11/wayland problem sorry for bothering you then. And i meant i haven't got time to take a good look at the projekt so i don't know and just assumed there is no script to restore the **fish** shell with theme for example 'cause there was _only_ one named _zsh_restore.sh_

**Reply by kRHYME7** [reply]
> Why is "screenshot" described "Frozen Screen"?

Dunno why "Frozen" but this means that before taking a screenshot the screen will freeze. It needs another dependency(hypicker ) to do that. Screenshots on Wayland WM are broken. 


> Brave(flatpak) and youtube-music-bin(aur) don't have an icon there but in applications menu they've. Why is "screenshot" described "Frozen Screen"?

Most of the time, installation from aur or the package itself don't provide the standards. (we're on Linux so what is standardization)

> 'cause there was only one named zsh_restore.sh

mostly the script is use to regenerate the plugin array on zshrc
